.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,36 @@
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - edited
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  schedule:
+    - cron: '0 0 * * *' # schedule daily check
```

Comment:
Could you move this later in the night pls :) ?
https://github.com/berty/go-libtor/blob/717006e09091206e862bb731f330fef2ea980724/.github/workflows/update.yaml#L5

Reply:
Done. Not sure why DCO bot is complaining now.